app/helpers/tasks_helper.rb

```diff
@@ -1,2 +1,9 @@
 module TasksHelper
+  def task_status(task)
```

Comment:
NIT:
If you have instance variable in controller for example `@task`, you don't have to pass task as argument but you can do this:
```ruby
# In helper:
  def task_status
    @task.status ? 'Done' : 'To Do'
  end

  def task_label
    @task.label&.title || 'None'
  end

# In views:
<%= task_status %>
<%= task_label %>
```